src/main/webapp/resources/js/pages/analysis/components/phylocanvas/LegendSection.tsx

```diff
@@ -0,0 +1,55 @@
+import { List, Space } from "antd";
+import React, { useMemo } from "react";
+import { LegendSectionItem } from "./LegendSectionItem";
+
+type LegendSectionProps = {
+  title: string;
+  sectionColourMap: Record<string, string>;
+  onSectionItemColourChange: (key: string, colour: string) => void;
+};
+
+const sortSection = (a: string, b: string) => {
```

Comment:
Probably would be a good idea to put this into a utilities file so we can use it elsewhere as needed

Reply:
Updated in 866c558